discordrb.json

```diff
@@ -57,6 +57,29 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
+            "userLoginWithToken": "yes",
+            "botLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "automatic",
+            "apiDocsUrl": "http://www.rubydoc.info/gems/discordrb",
+            "tutorials": "yes",
+            "tutorialsUrl": "https://github.com/meew0/discordrb/wiki",
+            "examples": "yes",
+            "examplesUrl": "https://github.com/meew0/discordrb/tree/master/examples",
+            "eventsAsync": "threaded",
+            "apiCallsAsync": "thread-safe",
+            "commandExtension": "yes",
+            "dcaExtension": "yes",
```

Comment:
@Rapptz DCA support is useful so people can pre-encode commonly played files and not have them encoded on the fly each time, reducing CPU usage. While supporting it is not a must, it would be nice to have and highlight libs that have it.